testsuite/domain/src/test/resources/host-configs/host-failover3.xml

```diff
@@ -55,7 +55,7 @@
     </management>
 
     <domain-controller>
-        <remote host="${jboss.test.host.master.address}" port="9999" security-realm="ManagementRealm">
+        <remote host="${jboss.test.host.master.address}" port="9999" security-realm="ManagementRealm" ignore-unused-configuration="false">
```

Comment:
Verified that these tests still pass after removal.

On Fri, Sep 11, 2015 at 11:51 AM, Ken Wills <ken@zaptillion.net> wrote:

> Ah, removing and retesting to verify.
>
> On Fri, Sep 11, 2015 at 11:47 AM, Brian Stansberry <
> notifications@github.com> wrote:
>
>> In testsuite/domain/src/test/resources/host-configs/host-failover3.xml
>> <https://github.com/wildfly/wildfly-core/pull/1057#discussion_r39290811>:
>>
>> > @@ -55,7 +55,7 @@
>> >      </management>
>> >
>> >      <domain-controller>
>> > -        <remote host="${jboss.test.host.master.address}" port="9999" security-realm="ManagementRealm">
>> > +        <remote host="${jboss.test.host.master.address}" port="9999" security-realm="ManagementRealm" ignore-unused-configuration="false">
>>
>> Same comment as for host-failover2.xml.
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/wildfly/wildfly-core/pull/1057/files#r39290811>.
>>
>
>